src/music.c

```diff
@@ -241,17 +241,6 @@ const char * SDLCALLCC Mix_GetMusicDecoder(int index)
     return(music_decoders[index]);
 }
 
-SDL_bool Mix_HasMusicDecoder(const char *name)
```

Comment:
Don't remove this call, it was added to prevent the link fault, that was done at mainstream SDL Mixer